src/test/java/edu/cornell/kfs/sys/rest/util/ApacheHttpJerseyExampleTest.java

```diff
@@ -0,0 +1,145 @@
+package edu.cornell.kfs.sys.rest.util;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.stream.Stream;
+
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.Response;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.parallel.Execution;
+import org.junit.jupiter.api.parallel.ExecutionMode;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.rest.CuJaxrsTestConstants.TestAppConstants;
+import edu.cornell.kfs.sys.rest.application.TestJaxrsApplication;
+import edu.cornell.kfs.sys.rest.resource.TestJaxrsResource;
+
+@Execution(ExecutionMode.SAME_THREAD)
+public class ApacheHttpJerseyExampleTest {
```

Comment:
Is this example needed?  It looks like ConcurAccessTokenServiceImplTest exercises ApacheHttpJerseyTestExtension

Reply:
As long as we expect at least one other test (like the Concur one) to remain around to exercise the ApacheHttpJerseyTestExtension code, I'm fine with removing this example test. I'll go ahead and remove this for now.

Reply:
Lets bring this up in the stand up tomorrow.  If there is a user story for the next usage of AWS secrete manager, a not should be added to make a unit test similar to this.

Reply:
Where do we stand with this?  I thought we wanted to remove this example since we have working tests for Concur?

Reply:
I agree that the Concur code is exercising enough of this functionality for now. I'll proceed with removing this particular test.